cypress/support/step-definitions/button-toggle-steps.js

```diff
@@ -68,3 +68,9 @@ Then('Button Toggle is not grouped', () => {
   buttonTogglePreview().eq(SECOND_ELEMENT).should('have.css', 'margin-left', '10px');
   buttonTogglePreview().eq(THIRD_ELEMENT).should('have.css', 'margin-left', '10px');
 });
+
+When('I click on Button Toggle', (element) => {
+  buttonTogglePreview(element).eq(FIRST_ELEMENT).click();
```

Comment:
As I mentioned above, if we need to check click event for every button we need to click on every button separately. And pass which one button we should click

Reply:
Amended to click on then check each button in a separate scenario example 👍